<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" rowspan="5">
     <widget class="QTreeView" name="treeView"/>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="newFolderButton">
      <property name="text">
       <string>New Folder</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="newFileButton">
      <property name="text">
       <string>New File</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Delete selected</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLogout"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAdministration">
    <property name="title">
     <string>Administration if admin</string>
    </property>
    <addaction name="actionEditUsers"/>
    <addaction name="actionLog"/>
    <addaction name="actionMonitor"/>
    <addaction name="actionProtectedObjects"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAdministration"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEditUsers">
   <property name="text">
    <string>Edit users</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionMonitor">
   <property name="text">
    <string>Controled files</string>
   </property>
  </action>
  <action name="actionProtectedObjects">
   <property name="text">
    <string>Edit protected objects</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
